rules-reviewed/jboss-eap/websphere/websphere-other.windup.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset id="websphere-other"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            Provides analysis of WebSphere proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="websphere" versionRange="[0,)"/>
+        <targetTechnology id="eap" versionRange="[6,)"/>
+        <targetTechnology id="java-ee" versionRange="[6,)"/>
+        <tag>reviewed-2015-06-09</tag>
+        <tag>ibm</tag>
+    </metadata>
+
+
+    <rules>
+        <rule id="websphere-other-01000">
+            <when>
+                <javaclass references="com.ibm.websphere.asynchbeans.Work">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification title="WebSphere Asynchronous Work"/>
+                <hint effort="3">
+                    <message>Migrate to JCA Work Manager.</message>
+                </hint>
+            </perform>
+        </rule>
+
+        <!-- Startup Service -->
+        <rule id="websphere-other-02000">
+            <when>
+                <javaclass references="com.ibm.websphere.startupservice.{*}StartUp{*}">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification title="WebSphere Startup Service" effort="4"/>
+                <hint effort="1">
```

Comment:
It appears that all of your hints need to be updated with MANDATORY/OPTIONAL as appropriate. I don't recall seeing any of them in this PR.